tests/system/common/clean_pi_system.py

```diff
@@ -54,8 +54,11 @@ def search_for_pi_point(host, admin, password, asset_name, data_point_name):
     except Exception:
         print("Could not get Points data.")
         return False, None
+    if data_point_name != '':
```

Comment:
Please add comments and Jira # reference as Known behaviour for single datapoint for an asset reading